gammapy/scripts/downloadclasses.py

```diff
@@ -5,9 +5,9 @@
 import logging
 import sys
 import yaml
-import multiprocessing
 from urllib.request import urlretrieve, urlopen
 from pathlib import Path
+from parfive import Downloader
```

Comment:
Suggest to make parfive an optional dependency, which I think means you have to make this a delayed import, and in the test put `@requires_dependency`?